src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+const sortList = (list) => {
+  function getNumber(salary) {
+    return +salary.slice(1).split(',').join('');
+  }
+
+  const sortedList = [...list].sort((a, b) => {
+    const aSalary = getNumber(a.dataset.salary);
+    const bSalary = getNumber(b.dataset.salary);
+
+    return bSalary - aSalary;
+  });
+
+  const ul = document.querySelector('ul');
+
+  sortedList.map(item => ul.appendChild(item));
+};
+
+const getEmployees = (list) => {
+  return [...list].map(item => item.dataset.name);
+};
```

Comment:
not just array of names, but objects
![image](https://user-images.githubusercontent.com/122989585/225561741-d95faa33-9f3b-445c-af69-694c19e5ec0f.png)